.changeset/light-ties-rhyme.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
changeset check is failing. Added this fie via `yarn changeset add`, but this seems incorrect. 🤔